core/src/main/scala/org/apache/spark/util/SizeEstimator.scala

```diff
@@ -283,4 +323,7 @@ private[spark] object SizeEstimator extends Logging {
     val rem = size % ALIGN_SIZE
     if (rem == 0) size else (size + ALIGN_SIZE - rem)
   }
+
+  private def alignSizeUp(size: Long, alignSize: Int): Long =
```

Comment:
I think this code is faster than the previous one and it's self-documentary. Do I need to replace the original one?